plugins/org.jkiss.dbeaver.ext.cubrid/plugin.xml

```diff
@@ -0,0 +1,233 @@
+	                            <items icon="#index" label="%tree.index.node.name" path="index" property="indexes" virtual="true">
+	                                <items icon="#column" label="%tree.index_columns.node.name" itemLabel="%tree.column.node.name" path="column" property="attributeReferences" inline="true" navigable="false" virtual="true">
+	                                </items>
+	                            </items>
+	                        </folder>
+	                        <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridProcedure" label="%tree.procedures.node.name" icon="#procedures" description="Procedures" visibleIf="object.dataSource.info.supportsStoredCode()">
+	                            <items label="%tree.procedures.node.name" itemLabel="%tree.procedure.node.name" path="procedure" property="proceduresOnly" icon="#procedure">
+	                                <folder label="%tree.procedure_columns.node.name" icon="#columns" description="Procedure parameters">
+	                                    <items label="%tree.procedure_columns.node.name" itemLabel="%tree.column.node.name" path="column" property="parameters" navigable="false"/>
+	                                </folder>
+	                            </items>
+	                            <items label="%tree.package.node.name" path="package" property="packages" icon="#package">
+	                                <items label="%tree.procedures.node.name" itemLabel="%tree.procedure.node.name" path="procedure" property="procedures" icon="#procedure">
+	                                    <items label="%tree.procedure_columns.node.name" itemLabel="%tree.column.node.name" path="column" property="parameters" navigable="false" inline="true"/>
+	                                </items>
+	                            </items>
+	                        </folder>
+	                	</items>
+                    </items>
+                    <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridUser" label="%tree.users.node.name" icon="#sequences" description="%tree.sequences.node.tip">
+                        <items label="%tree.user.node.name" path="cubridUser" property="cubridUsers" icon="#user"/>
+                    </folder>
+             		<folder type="org.jkiss.dbeaver.model.struct.DBSDataType" label="%tree.dataTypes.node.name" icon="#data_types" description="%tree.dataTypes.node.tip">
+                        <items label="%tree.dataType.node.name" path="dataType" property="dataTypes" icon="#data_type"/>
+                    </folder>
+                </items>
+            </tree>
+            
+            <drivers managable="true">
+                <driver
+                        id="cubrid_jdbc"
+                        label="CUBRID"
+                        icon="icons/cubrid_icon.png"

```

Comment:
The Cubrid icon is not displayed properly, need to check if this is related to this.
maybe Is the image the problem?
![image](https://github.com/CUBRID/dbeaver/assets/68524199/bfcf735b-09be-43f3-845c-9b3bea848e81)

Reply:
Hello @hwany7seo, 

For this issue, I have tried to clone it to my local and test it with the new workspace in Eclipse, it works pretty fine. I am not sure whether this problem is related to Eclipse or not. If possible, Could you test it again with the new workspace?

![image](https://github.com/CUBRID/dbeaver/assets/123133969/a3b99f16-3cb9-43f4-a564-997bee7640f9)

Reply:
Hello @longhaseng52 .
All testing should be done output package on mvn project.
The png file is missing because the icons folder is missing from 'build.properties'.
![image](https://github.com/CUBRID/dbeaver/assets/68524199/70e7b298-ef5a-4a30-bd43-fc3214cb1333)

Reply:
I see, I will update it now. Thank you